code/modules/holidays/halloween/pumpkins/patches.dm

```diff
@@ -0,0 +1,58 @@
+/// Patches of pumpkins spawned at roundstart from where marines can get their carvable pumpkins
+/obj/structure/pumpkin_patch
+	icon = 'icons/misc/events/pumpkins.dmi'
+	name = "patch of pumpkins"
+
+	can_block_movement = FALSE
+	unslashable = TRUE
+	health = 400 // To avoid explosions and stray gunfire destroying them too easily
+	layer = LOWER_ITEM_LAYER
+
+	var/has_vines = TRUE //! Whether there's still vines to display or not
+	var/pumpkin_count = 3 //! Amount of pumpkins currently in the patch
+	var/icon_prefix //! Prefix to prepend to icon states, for corrupted pumpkins
+	var/pumpkin_type = /obj/item/clothing/head/pumpkin
+
+/obj/structure/pumpkin_patch/Initialize(mapload, ...)
+	. = ..()
+	update_icon()
+
+/obj/structure/pumpkin_patch/update_icon()
+	overlays?.Cut()
+	. = ..()
+	switch(pumpkin_count)
+		if(3) icon_state = "[icon_prefix]pumpkins_full"
+		if(2) icon_state = "[icon_prefix]pumpkins_half"
+		if(1) icon_state = "[icon_prefix]pumpkin"
+		else  icon_state = ""
```

Comment:
No pumpkins left leaves it as a placeholder examine image: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/0f370acb-d48e-47e5-989c-1a65ca7983cb)

Reply:
added a proper placeholder blank sprite